vmaas_sync/refresh.go

```diff
@@ -0,0 +1,27 @@
+package vmaas_sync //nolint:golint,stylecheck
+
+import (
+	"app/base/utils"
+	"gorm.io/gorm"
+	"time"
+)
+
+func refreshLatestPackagesCount() {
+	defer utils.LogPanics(true)
+
+	ticker := time.NewTicker(time.Minute * 10)
+
+	for {
+		<-ticker.C
+
```

Comment:
Please, could you add env. var. option to enable/disable this job as done for other vmaas-sync jobs? It allows to disable only this part in case of any issues. Otherwise we would need to stop the whole component.